temperature/crud.py

```diff
@@ -0,0 +1,54 @@
+import datetime
+
+from sqlalchemy import select
+from sqlalchemy.orm import selectinload
+
+from city.crud import get_all_cities
+from temperature import models, schemas
+
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from temperature.utils import get_temperature
+
+
+async def update_temperatures(db: AsyncSession) -> dict:
+    cities = await get_all_cities(db=db)
```

Comment:
The `get_all_cities` function is used here to fetch all cities. Ensure that this function returns the expected list of city objects with the necessary attributes for further processing.